app/services/geek_jokes.js

```diff
@@ -0,0 +1,9 @@
+const axios = require('axios');
+const config = require('../../config');
+
+const { url } = config && config.common && config.common.witter_api;
+const getGeekJoke = () => axios.get(url);
```

Comment:
It is very helpful to log the requests and responses, you can use this helper to do that: `app/logger`